bake/entitlements_windows.go

```diff
@@ -10,7 +10,7 @@ import (
 func evaluatePaths(in []string) ([]string, error) {
 	out := make([]string, 0, len(in))
 	for _, p := range in {
-		if p == "/" {
+		if p == "*" {
```

Comment:
Introduce wildcard pattern to allow any volumes on windows. On unix it would be `/`.

Reply:
I think we should just handle `*` as a special value that acts as a valid parent for any path, rather than try to convert `*` into a list of all possible volumes on windows.